core/pkg/policyhandler/handlenotification.go

```diff
@@ -45,16 +47,30 @@ func (policyHandler *PolicyHandler) CollectResources(notification *reporthandlin
 	return opaSessionObj, nil
 }
 
-func (policyHandler *PolicyHandler) getResources(notification *reporthandling.PolicyNotification, opaSessionObj *cautils.OPASessionObj, scanInfo *cautils.ScanInfo) error {
+func setOpaSessionObj(opaSessionObj *cautils.OPASessionObj, scanInfo *cautils.ScanInfo) {
```

Comment:
function name should be: `scanInfoToScanMetadata`

Notice, you set the values of metadata